### ISSUE TITLE
Possible to activate multiple prop-mods

### ISSUE THREAD

**Issue by Kalaratiri** [pyfa-org/Pyfa#1720]
## Bug Report
More than one propulsion module can be activated/does not automatically deactivate the first prop-mod on adding the second

### Expected behavior:
Fitting an mwd while an afterburner is fitted should turn the afterburner into the "off" state. 
It should not be possible to activate one type of prop-mod while another is already activated.
It should not be possible to activate two of the same prop-mod.

### Actual behavior:
Fitting a second propulsion module does not deactivate the one already fitted.
It is possible to have both an afterburner and an mwd active on the fit. https://i.imgur.com/E9HYM1i.png
It is possible to have two of the same propulsion modules active on the fit. https://i.imgur.com/7F7Eour.png

### Detailed steps to reproduce:
Fit any two prop-mods.

### Fits involved in EFT format (Edit > To Clipboard > EFT):
Any fit.

### Release or development git branch? Please note the release version or commit hash:
2.5.0b1

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
Amusingly, the speed appears to be capped at 1,000,000m/s, although an 8 mwd scorpion can get considerably faster than this. https://i.imgur.com/1MchmhQ.png

**Comment by blitzmann** [comment]
Merp. Yeah, this is a thing. A byproduct of the new Command pattern that enables undo/redo.

The reasoning behind it is due to the fact that, before, when we added a module, we were able to add it then run it through a `checkStates` function, which checks the validity of states. If we found an invalid states, we would turn it off (based on another module).

We can no longer do that with the Command pattern because we don't want to do a recalc within the command. The reasoning behind this is because it's not part of the command's scope - there may be situations where we need to do three or four commands, and we don't want to recalc after each one (usually), so the recalc happens at a higher level than the command itself.

I meant to look into this bug and find a better way of fixing states - one that doesn't require a recalculation. I don't think it would be difficult, just never got around to actually looking into it.

All in all, I think this is a minor bug in comparison to the usefulness of what we get with the command pattern (undo/redo). If it needs to live around for a while until it gets fixed, I'm okay with that. That being said, I will try to look into this issue again before the actual release.

😄
